e2e-tests/gatsby-static-image/cypress/integration/native-lazy-loading.js

```diff
@@ -1,7 +1,8 @@
 describe(`gatsby-plugin-image / native lazy loading`, () => {
-  // /!\ Make sure to run this one using a real build: not in develop
-  // because we need SSR rendering in that scenario
   beforeEach(() => {
+    // /!\ Make sure to run this one using a real build: not in develop
+    // because we need SSR rendering in that scenario
+    cy.wrap(Cypress.config().baseUrl).should(`contain`, `:9000`)
```

Comment:
I'd check that it's _not_ 8000, rather than that it is 9000, so that it can be run against a liive site if needed